### PR TITLE
Add AddCommand, DeleteCommand, CommandResult, enum in Task

### LINKED ISSUE #14: As a user, I want to add tasks to the scheduler so that I can keep track of them
Add a task: add -t [-by]
Add a task to the scheduler
Format: add -t <task_name> [-by <deadline>]

### LINKED ISSUE #15: As a user, I want to add deadlines for tasks so that I know when I need to complete them
Add a task: add -t [-by]
Add a task to the scheduler
Format: add -t <task_name> [-by <deadline>]

### LINKED ISSUE #16: As a user, I want to add modules to the scheduler so that I can plan my timetable quickly
Add a module: add -m
Format: add -m <module_code>

### LINKED ISSUE #30: As a user, I want to be able to delete modules so that I can plan my timetable quickly.

### PR DESCRIPTION
**Pull request by Aseanseen**

### CODE REVIEW

File: src/main/java/seedu/duke/command/CommandResult.java [hunk 1]
```diff
@@ -6,8 +6,4 @@
     public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
     }
-
-    public String getFeedbackToUser() {
```
Comment (SimJunYou) [inline src/main/java/seedu/duke/command/CommandResult.java:10]: If removing getter function, then the `feedbackToUser` attribute should not be private.

**Review by SimJunYou — CHANGES_REQUESTED** [review]
Looks fine overall, just one thing to change I think.

**Comment by Aseanseen** [comment]
Amended the AddCommand, DeleteCommand, enum in Parser. Checks are failing due to the functions it requires from other classes.

AddCommand:
testDeadline(String deadline)
addTask(String desc, LocalDateTime dateTimeOfDeadline)
addModule(String desc)
execute()

DeleteCommand:
deleteTask(int taskId)
deleteModule(String moduleCode)
execute()

**Review by SimJunYou — APPROVED** [review]
Looks good, let's go

**Comment by tobiasceg** [comment]
could u add, a AddCommand.FORMAT
a constant that outputs the correct format on how a addcommand will be read

File: src/main/java/seedu/duke/command/add/AddCommand.java [hunk 2]
```diff
@@ -1,6 +1,98 @@
...
+import seedu.duke.parser.Parser;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import static seedu.duke.util.Message.MESSAGE_ADD_MODULE_SUCCESS;
+import static seedu.duke.util.Message.MESSAGE_ADD_TASK_SUCCESS;
+import static seedu.duke.util.ExceptionMessage.MESSAGE_DUPLICATE_MODULE;
 
-public abstract class AddCommand extends Command {
+public class AddCommand extends Command {
```
Comment (tobiasceg) [inline src/main/java/seedu/duke/command/add/AddCommand.java:18]: Add a constant FORMAT

**Comment by Aseanseen** [comment]
> could u add, a AddCommand.FORMAT
> a constant that outputs the correct format on how a addcommand will be read

Ok, added

**Pull request merged**